src/sentry_plugins/clubhouse/client.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from sentry_plugins.client import ApiClient
+
+
+class ClubhouseClient(ApiClient):
+    base_url = u'https://api.clubhouse.io/api/v2'
+
+    # OPTIMIZE: Override ApiClient.request to add the token param, to simplify individual calls.
+    # Needs to receive a token value on instantiation somehow.
+    # def request(self, *args, **kwargs):
+    #     params={'token': token}
+    #     return self._request(*args, **kwargs)
+
+    def get_story(self, story_id, token):
+        return self.get('/stories/%s' % story_id, params={'token': token})
+
+    def create_story(self, project, data, token):
+        story_data = {
+            'project_id': project,
+            'name': data['title'],
+            'description': data['description'],
```

Comment:
Discuss: Should this automatically add `'story_type': 'bug'` to Stories created by users from Sentry?  Currently defaults to `feature`.

Reply:
+1

Reply:
Great! I'll implement the change.

Reply:
Unfortunately, since the above line hasn't changed, GitHub won't collapse this comment stream despite it being implemented.